lib/Service/Appointments/EventConflictFilter.php

```diff
@@ -56,14 +60,31 @@ public function filter(AppointmentConfig $config, array $slots): array {
 		}
 		// Always check the target calendar for conflicts
 		$query->addSearchCalendar($config->getTargetCalendarUri());
+		$query->addType('VEVENT');
 		$preparationDuration = DateInterval::createFromDateString($config->getPreparationDuration() . ' seconds');
 		$followUpDuration = DateInterval::createFromDateString($config->getFollowupDuration() . ' seconds');
-
+		$this->logger->debug('Appointment config ' . $config->getToken() . ' is looking with preparation duration {preparation} and follow up duration {followup} in calendar {calendarUri}', [
+			'preparation' => $preparationDuration->s,
```

Comment:
It would be nice if we could have BookingCalendarWriter::secondsToIso8601Duration in a helper class and call it here instead of giving the number of seconds that don't make a lot of sense.